test/unit/local/steps/initial_diff.js

```diff
@@ -26,10 +26,12 @@ describe('local/steps/initial_diff.loop()', () => {
     await builders.metadir().path('foo').ino(1).create()
     await builders.metafile().path('fizz').ino(2).create()
 
+    const state = await initialDiff.initialState({ pouch: this.pouch })
```

Comment:
DRY? (`beforeEach` hook)

Reply:
It needs to happen after we've created the metatada documents. So unless we can move those create in a beforeEach hook (which I believe we can't) then we need to have that call in the test itself.